source/install.txt

```diff
@@ -172,7 +172,17 @@ To download and install |compass|, select your operating system:
 
                .. code-block:: shell
 
-                  sudo dpkg -i mongodb-compass_{+current-version+}_amd64.deb
+                  sudo apt install mongodb-compass_{+current-version+}_amd64.deb
+                  
+               .. note::
```

Comment:
(optional suggestion) 
This is subjective. I think this page might be getting a little note heavy. What do you think about making it just a second paragraph?

Reply:
Done! I also added punctuation to the steps on this page, thanks for your feedback!